packages/notebook/src/browser/view-model/notebook-model.ts

```diff
@@ -71,6 +71,15 @@ export class NotebookModel implements Saveable, Disposable {
     @inject(UndoRedoService)
     private readonly undoRedoService: UndoRedoService;
 
+    @inject(NotebookModelProps)
+    private props: NotebookModelProps;
```

Comment:
Suggestion: All of these injected items and the event emitters should be `protected`.